.github/workflows/release.yml

```diff
@@ -41,27 +41,27 @@ on:
         default: INFO
         required: false 
         description: "Sets the logging level to get more detailed info."  
-      release_bundle_name:
-        description: "Release Bundle Name"
-        required: false
-        type: string
-      release_bundle_version:
-        description: "Release Bundle Version"
-        required: false
-        type: string 
-      signing_key_name:
-        description: "The GPG/RSA key-pair name given in Artifactory."
-        required: false
-        type: string 
-      release_method:
-        description: "Source type to create release bundle (i.e builds or artifacts)."
-        required: false
-        default: artifacts
-        type: string       
+      # release_bundle_name:
```

Comment:
If these params are not needed, don't leave commented code in here.

Reply:
removed commented code